tests/crud/test_crud_endpoints.py

```diff
@@ -493,33 +511,31 @@ class Rating(Enum):
         self.assertEqual(
             response.json(),
             {
-                "title": "ReviewIn",
-                "type": "object",
+                "help_text": None,
+                "primary_key_name": "id",
```

Comment:
So most of these changes are because Pydantic v2 changed the order of fields in the response?

Reply:
Yes. In Pydantic v2 the schema fields are sorted in alphabetical order.